curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/6352f179bdca23221298a5ba.md

```diff
@@ -0,0 +1,185 @@
+---
+id: 6352f179bdca23221298a5ba
+title: Step 35
+challengeType: 0
+dashedName: step-35
+---
+
+# --description--
+
+Your next calculation is the <dfn>range</dfn>, which is the difference between the largest and smallest numbers in the list.
+
+Declare a `getRange` function that takes the same `array` parameter you have been using. Using `Math.min`, `Math.max`, and the spread operator, return the difference between the largest and smallest numbers in the list.
```

Comment:
Have we covered Math.min or max yet?
If not, maybe we could add a small example.